Automattic/woocommerce-payments
# refactor: loading indicator on payments section

Fixes #

#### Changes proposed in this Pull Request









I'm thinking that using a pseudo element might be more evident, rather than animating the background of the payments section.
Otherwise the payment methods seem to be on top of the background, and not part of the animation.
The pseudo element also ensures that the user cannot interact with the buttons with the mouse.

Before:
![2021-07-14 12 34 45](https://user-images.githubusercontent.com/273592/125667265-f916d74a-b0d6-4bc2-9114-ac72f0e2d0a6.gif)

After:
![2021-07-14 12 33 55](https://user-images.githubusercontent.com/273592/125667299-f34c0c98-d8f7-4a8f-8f48-3e0eff7ad299.gif)


#### Testing instructions





- Ensure you have "Enable UPE settings changes" enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Enable/disable UPE to see the animation

-------------------
